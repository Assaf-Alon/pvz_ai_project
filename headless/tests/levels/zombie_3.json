{
    "10": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]],
    "20": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]],
    "30": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]],
    "40": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]],
    "50": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]],
    "60": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]],
    "70": [["Normal", 0], ["Normal", 1], ["Normal", 2], ["Normal", 3], ["Normal", 4]]
}
