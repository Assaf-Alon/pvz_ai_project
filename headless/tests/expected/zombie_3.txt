[100] Normal zombie being spawned at lane 0
[100] Normal zombie being spawned at lane 1
[100] Normal zombie being spawned at lane 2
[100] Normal zombie being spawned at lane 3
[100] Normal zombie being spawned at lane 4
[100] Sun autospawned. Total: 75.
[147] Normal zombie in (0, 9) moved to (0, 8).
[147] Normal zombie in (1, 9) moved to (1, 8).
[147] Normal zombie in (2, 9) moved to (2, 8).
[147] Normal zombie in (3, 9) moved to (3, 8).
[147] Normal zombie in (4, 9) moved to (4, 8).
[194] Normal zombie in (0, 8) moved to (0, 7).
[194] Normal zombie in (1, 8) moved to (1, 7).
[194] Normal zombie in (2, 8) moved to (2, 7).
[194] Normal zombie in (3, 8) moved to (3, 7).
[194] Normal zombie in (4, 8) moved to (4, 7).
[200] Normal zombie being spawned at lane 0
[200] Normal zombie being spawned at lane 1
[200] Normal zombie being spawned at lane 2
[200] Normal zombie being spawned at lane 3
[200] Normal zombie being spawned at lane 4
[200] Sun autospawned. Total: 100.
[241] Normal zombie in (0, 7) moved to (0, 6).
[241] Normal zombie in (1, 7) moved to (1, 6).
[241] Normal zombie in (2, 7) moved to (2, 6).
[241] Normal zombie in (3, 7) moved to (3, 6).
[241] Normal zombie in (4, 7) moved to (4, 6).
[247] Normal zombie in (0, 9) moved to (0, 8).
[247] Normal zombie in (1, 9) moved to (1, 8).
[247] Normal zombie in (2, 9) moved to (2, 8).
[247] Normal zombie in (3, 9) moved to (3, 8).
[247] Normal zombie in (4, 9) moved to (4, 8).
[288] Normal zombie in (0, 6) moved to (0, 5).
[288] Normal zombie in (1, 6) moved to (1, 5).
[288] Normal zombie in (2, 6) moved to (2, 5).
[288] Normal zombie in (3, 6) moved to (3, 5).
[288] Normal zombie in (4, 6) moved to (4, 5).
[294] Normal zombie in (0, 8) moved to (0, 7).
[294] Normal zombie in (1, 8) moved to (1, 7).
[294] Normal zombie in (2, 8) moved to (2, 7).
[294] Normal zombie in (3, 8) moved to (3, 7).
[294] Normal zombie in (4, 8) moved to (4, 7).
[300] Normal zombie being spawned at lane 0
[300] Normal zombie being spawned at lane 1
[300] Normal zombie being spawned at lane 2
[300] Normal zombie being spawned at lane 3
[300] Normal zombie being spawned at lane 4
[300] Sun autospawned. Total: 125.
[335] Normal zombie in (0, 5) moved to (0, 4).
[335] Normal zombie in (1, 5) moved to (1, 4).
[335] Normal zombie in (2, 5) moved to (2, 4).
[335] Normal zombie in (3, 5) moved to (3, 4).
[335] Normal zombie in (4, 5) moved to (4, 4).
[341] Normal zombie in (0, 7) moved to (0, 6).
[341] Normal zombie in (1, 7) moved to (1, 6).
[341] Normal zombie in (2, 7) moved to (2, 6).
[341] Normal zombie in (3, 7) moved to (3, 6).
[341] Normal zombie in (4, 7) moved to (4, 6).
[347] Normal zombie in (0, 9) moved to (0, 8).
[347] Normal zombie in (1, 9) moved to (1, 8).
[347] Normal zombie in (2, 9) moved to (2, 8).
[347] Normal zombie in (3, 9) moved to (3, 8).
[347] Normal zombie in (4, 9) moved to (4, 8).
[382] Normal zombie in (0, 4) moved to (0, 3).
[382] Normal zombie in (1, 4) moved to (1, 3).
[382] Normal zombie in (2, 4) moved to (2, 3).
[382] Normal zombie in (3, 4) moved to (3, 3).
[382] Normal zombie in (4, 4) moved to (4, 3).
[388] Normal zombie in (0, 6) moved to (0, 5).
[388] Normal zombie in (1, 6) moved to (1, 5).
[388] Normal zombie in (2, 6) moved to (2, 5).
[388] Normal zombie in (3, 6) moved to (3, 5).
[388] Normal zombie in (4, 6) moved to (4, 5).
[394] Normal zombie in (0, 8) moved to (0, 7).
[394] Normal zombie in (1, 8) moved to (1, 7).
[394] Normal zombie in (2, 8) moved to (2, 7).
[394] Normal zombie in (3, 8) moved to (3, 7).
[394] Normal zombie in (4, 8) moved to (4, 7).
[400] Normal zombie being spawned at lane 0
[400] Normal zombie being spawned at lane 1
[400] Normal zombie being spawned at lane 2
[400] Normal zombie being spawned at lane 3
[400] Normal zombie being spawned at lane 4
[400] Sun autospawned. Total: 150.
[429] Normal zombie in (0, 3) moved to (0, 2).
[429] Normal zombie in (1, 3) moved to (1, 2).
[429] Normal zombie in (2, 3) moved to (2, 2).
[429] Normal zombie in (3, 3) moved to (3, 2).
[429] Normal zombie in (4, 3) moved to (4, 2).
[435] Normal zombie in (0, 5) moved to (0, 4).
[435] Normal zombie in (1, 5) moved to (1, 4).
[435] Normal zombie in (2, 5) moved to (2, 4).
[435] Normal zombie in (3, 5) moved to (3, 4).
[435] Normal zombie in (4, 5) moved to (4, 4).
[441] Normal zombie in (0, 7) moved to (0, 6).
[441] Normal zombie in (1, 7) moved to (1, 6).
[441] Normal zombie in (2, 7) moved to (2, 6).
[441] Normal zombie in (3, 7) moved to (3, 6).
[441] Normal zombie in (4, 7) moved to (4, 6).
[447] Normal zombie in (0, 9) moved to (0, 8).
[447] Normal zombie in (1, 9) moved to (1, 8).
[447] Normal zombie in (2, 9) moved to (2, 8).
[447] Normal zombie in (3, 9) moved to (3, 8).
[447] Normal zombie in (4, 9) moved to (4, 8).
[476] Normal zombie in (0, 2) moved to (0, 1).
[476] Normal zombie in (1, 2) moved to (1, 1).
[476] Normal zombie in (2, 2) moved to (2, 1).
[476] Normal zombie in (3, 2) moved to (3, 1).
[476] Normal zombie in (4, 2) moved to (4, 1).
[482] Normal zombie in (0, 4) moved to (0, 3).
[482] Normal zombie in (1, 4) moved to (1, 3).
[482] Normal zombie in (2, 4) moved to (2, 3).
[482] Normal zombie in (3, 4) moved to (3, 3).
[482] Normal zombie in (4, 4) moved to (4, 3).
[488] Normal zombie in (0, 6) moved to (0, 5).
[488] Normal zombie in (1, 6) moved to (1, 5).
[488] Normal zombie in (2, 6) moved to (2, 5).
[488] Normal zombie in (3, 6) moved to (3, 5).
[488] Normal zombie in (4, 6) moved to (4, 5).
[494] Normal zombie in (0, 8) moved to (0, 7).
[494] Normal zombie in (1, 8) moved to (1, 7).
[494] Normal zombie in (2, 8) moved to (2, 7).
[494] Normal zombie in (3, 8) moved to (3, 7).
[494] Normal zombie in (4, 8) moved to (4, 7).
[500] Normal zombie being spawned at lane 0
[500] Normal zombie being spawned at lane 1
[500] Normal zombie being spawned at lane 2
[500] Normal zombie being spawned at lane 3
[500] Normal zombie being spawned at lane 4
[500] Sun autospawned. Total: 175.
[523] Normal zombie in (0, 1) moved to (0, 0).
[523] Normal zombie in (1, 1) moved to (1, 0).
[523] Normal zombie in (2, 1) moved to (2, 0).
[523] Normal zombie in (3, 1) moved to (3, 0).
[523] Normal zombie in (4, 1) moved to (4, 0).
[529] Normal zombie in (0, 3) moved to (0, 2).
[529] Normal zombie in (1, 3) moved to (1, 2).
[529] Normal zombie in (2, 3) moved to (2, 2).
[529] Normal zombie in (3, 3) moved to (3, 2).
[529] Normal zombie in (4, 3) moved to (4, 2).
[535] Normal zombie in (0, 5) moved to (0, 4).
[535] Normal zombie in (1, 5) moved to (1, 4).
[535] Normal zombie in (2, 5) moved to (2, 4).
[535] Normal zombie in (3, 5) moved to (3, 4).
[535] Normal zombie in (4, 5) moved to (4, 4).
[541] Normal zombie in (0, 7) moved to (0, 6).
[541] Normal zombie in (1, 7) moved to (1, 6).
[541] Normal zombie in (2, 7) moved to (2, 6).
[541] Normal zombie in (3, 7) moved to (3, 6).
[541] Normal zombie in (4, 7) moved to (4, 6).
[547] Normal zombie in (0, 9) moved to (0, 8).
[547] Normal zombie in (1, 9) moved to (1, 8).
[547] Normal zombie in (2, 9) moved to (2, 8).
[547] Normal zombie in (3, 9) moved to (3, 8).
[547] Normal zombie in (4, 9) moved to (4, 8).
[570] Normal zombie in (0, 0) attempted entering the house.
[570] Normal zombie in (1, 0) attempted entering the house.
[570] Normal zombie in (2, 0) attempted entering the house.
[570] Normal zombie in (3, 0) attempted entering the house.
[570] Normal zombie in (4, 0) attempted entering the house.
[570] Normal zombie in (0, 0) triggered a lawnmower.
[570] Lawnmower in (0, 0) attacked.
[570] Normal zombie in (0, 0) was damaged. HP: -9818.
[570] Normal zombie in (0, 0) was killed.
[570] Normal zombie in (1, 0) triggered a lawnmower.
[570] Lawnmower in (1, 0) attacked.
[570] Normal zombie in (1, 0) was damaged. HP: -9818.
[570] Normal zombie in (1, 0) was killed.
[570] Normal zombie in (2, 0) triggered a lawnmower.
[570] Lawnmower in (2, 0) attacked.
[570] Normal zombie in (2, 0) was damaged. HP: -9818.
[570] Normal zombie in (2, 0) was killed.
[570] Normal zombie in (3, 0) triggered a lawnmower.
[570] Lawnmower in (3, 0) attacked.
[570] Normal zombie in (3, 0) was damaged. HP: -9818.
[570] Normal zombie in (3, 0) was killed.
[570] Normal zombie in (4, 0) triggered a lawnmower.
[570] Lawnmower in (4, 0) attacked.
[570] Normal zombie in (4, 0) was damaged. HP: -9818.
[570] Normal zombie in (4, 0) was killed.
[571] Lawnmower in (0, 0) moved to (0, 1).
[571] Lawnmower in (1, 0) moved to (1, 1).
[571] Lawnmower in (2, 0) moved to (2, 1).
[571] Lawnmower in (3, 0) moved to (3, 1).
[571] Lawnmower in (4, 0) moved to (4, 1).
[573] Lawnmower in (0, 1) moved to (0, 2).
[573] Lawnmower in (0, 2) attacked.
[573] Normal zombie in (0, 2) was damaged. HP: -9818.
[573] Normal zombie in (0, 2) was killed.
[573] Lawnmower in (1, 1) moved to (1, 2).
[573] Lawnmower in (1, 2) attacked.
[573] Normal zombie in (1, 2) was damaged. HP: -9818.
[573] Normal zombie in (1, 2) was killed.
[573] Lawnmower in (2, 1) moved to (2, 2).
[573] Lawnmower in (2, 2) attacked.
[573] Normal zombie in (2, 2) was damaged. HP: -9818.
[573] Normal zombie in (2, 2) was killed.
[573] Lawnmower in (3, 1) moved to (3, 2).
[573] Lawnmower in (3, 2) attacked.
[573] Normal zombie in (3, 2) was damaged. HP: -9818.
[573] Normal zombie in (3, 2) was killed.
[573] Lawnmower in (4, 1) moved to (4, 2).
[573] Lawnmower in (4, 2) attacked.
[573] Normal zombie in (4, 2) was damaged. HP: -9818.
[573] Normal zombie in (4, 2) was killed.
[575] Lawnmower in (0, 2) moved to (0, 3).
[575] Lawnmower in (1, 2) moved to (1, 3).
[575] Lawnmower in (2, 2) moved to (2, 3).
[575] Lawnmower in (3, 2) moved to (3, 3).
[575] Lawnmower in (4, 2) moved to (4, 3).
[577] Lawnmower in (0, 3) moved to (0, 4).
[577] Lawnmower in (0, 4) attacked.
[577] Normal zombie in (0, 4) was damaged. HP: -9818.
[577] Normal zombie in (0, 4) was killed.
[577] Lawnmower in (1, 3) moved to (1, 4).
[577] Lawnmower in (1, 4) attacked.
[577] Normal zombie in (1, 4) was damaged. HP: -9818.
[577] Normal zombie in (1, 4) was killed.
[577] Lawnmower in (2, 3) moved to (2, 4).
[577] Lawnmower in (2, 4) attacked.
[577] Normal zombie in (2, 4) was damaged. HP: -9818.
[577] Normal zombie in (2, 4) was killed.
[577] Lawnmower in (3, 3) moved to (3, 4).
[577] Lawnmower in (3, 4) attacked.
[577] Normal zombie in (3, 4) was damaged. HP: -9818.
[577] Normal zombie in (3, 4) was killed.
[577] Lawnmower in (4, 3) moved to (4, 4).
[577] Lawnmower in (4, 4) attacked.
[577] Normal zombie in (4, 4) was damaged. HP: -9818.
[577] Normal zombie in (4, 4) was killed.
[579] Lawnmower in (0, 4) moved to (0, 5).
[579] Lawnmower in (1, 4) moved to (1, 5).
[579] Lawnmower in (2, 4) moved to (2, 5).
[579] Lawnmower in (3, 4) moved to (3, 5).
[579] Lawnmower in (4, 4) moved to (4, 5).
[581] Lawnmower in (0, 5) moved to (0, 6).
[581] Lawnmower in (0, 6) attacked.
[581] Normal zombie in (0, 6) was damaged. HP: -9818.
[581] Normal zombie in (0, 6) was killed.
[581] Lawnmower in (1, 5) moved to (1, 6).
[581] Lawnmower in (1, 6) attacked.
[581] Normal zombie in (1, 6) was damaged. HP: -9818.
[581] Normal zombie in (1, 6) was killed.
[581] Lawnmower in (2, 5) moved to (2, 6).
[581] Lawnmower in (2, 6) attacked.
[581] Normal zombie in (2, 6) was damaged. HP: -9818.
[581] Normal zombie in (2, 6) was killed.
[581] Lawnmower in (3, 5) moved to (3, 6).
[581] Lawnmower in (3, 6) attacked.
[581] Normal zombie in (3, 6) was damaged. HP: -9818.
[581] Normal zombie in (3, 6) was killed.
[581] Lawnmower in (4, 5) moved to (4, 6).
[581] Lawnmower in (4, 6) attacked.
[581] Normal zombie in (4, 6) was damaged. HP: -9818.
[581] Normal zombie in (4, 6) was killed.
[583] Lawnmower in (0, 6) moved to (0, 7).
[583] Lawnmower in (1, 6) moved to (1, 7).
[583] Lawnmower in (2, 6) moved to (2, 7).
[583] Lawnmower in (3, 6) moved to (3, 7).
[583] Lawnmower in (4, 6) moved to (4, 7).
[585] Lawnmower in (0, 7) moved to (0, 8).
[585] Lawnmower in (0, 8) attacked.
[585] Normal zombie in (0, 8) was damaged. HP: -9818.
[585] Normal zombie in (0, 8) was killed.
[585] Lawnmower in (1, 7) moved to (1, 8).
[585] Lawnmower in (1, 8) attacked.
[585] Normal zombie in (1, 8) was damaged. HP: -9818.
[585] Normal zombie in (1, 8) was killed.
[585] Lawnmower in (2, 7) moved to (2, 8).
[585] Lawnmower in (2, 8) attacked.
[585] Normal zombie in (2, 8) was damaged. HP: -9818.
[585] Normal zombie in (2, 8) was killed.
[585] Lawnmower in (3, 7) moved to (3, 8).
[585] Lawnmower in (3, 8) attacked.
[585] Normal zombie in (3, 8) was damaged. HP: -9818.
[585] Normal zombie in (3, 8) was killed.
[585] Lawnmower in (4, 7) moved to (4, 8).
[585] Lawnmower in (4, 8) attacked.
[585] Normal zombie in (4, 8) was damaged. HP: -9818.
[585] Normal zombie in (4, 8) was killed.
[587] Lawnmower in (0, 8) moved to (0, 9).
[587] Lawnmower in (1, 8) moved to (1, 9).
[587] Lawnmower in (2, 8) moved to (2, 9).
[587] Lawnmower in (3, 8) moved to (3, 9).
[587] Lawnmower in (4, 8) moved to (4, 9).
[589] Lawnmower in (0, 9) moved to (0, 10).
[589] Lawnmower in (0, 10) removed.
[589] Lawnmower in (1, 9) moved to (1, 10).
[589] Lawnmower in (1, 10) removed.
[589] Lawnmower in (2, 9) moved to (2, 10).
[589] Lawnmower in (2, 10) removed.
[589] Lawnmower in (3, 9) moved to (3, 10).
[589] Lawnmower in (3, 10) removed.
[589] Lawnmower in (4, 9) moved to (4, 10).
[589] Lawnmower in (4, 10) removed.
[600] Normal zombie being spawned at lane 0
[600] Normal zombie being spawned at lane 1
[600] Normal zombie being spawned at lane 2
[600] Normal zombie being spawned at lane 3
[600] Normal zombie being spawned at lane 4
[600] Sun autospawned. Total: 200.
[647] Normal zombie in (0, 9) moved to (0, 8).
[647] Normal zombie in (1, 9) moved to (1, 8).
[647] Normal zombie in (2, 9) moved to (2, 8).
[647] Normal zombie in (3, 9) moved to (3, 8).
[647] Normal zombie in (4, 9) moved to (4, 8).
[694] Normal zombie in (0, 8) moved to (0, 7).
[694] Normal zombie in (1, 8) moved to (1, 7).
[694] Normal zombie in (2, 8) moved to (2, 7).
[694] Normal zombie in (3, 8) moved to (3, 7).
[694] Normal zombie in (4, 8) moved to (4, 7).
[700] Normal zombie being spawned at lane 0
[700] Normal zombie being spawned at lane 1
[700] Normal zombie being spawned at lane 2
[700] Normal zombie being spawned at lane 3
[700] Normal zombie being spawned at lane 4
[700] Sun autospawned. Total: 225.
[741] Normal zombie in (0, 7) moved to (0, 6).
[741] Normal zombie in (1, 7) moved to (1, 6).
[741] Normal zombie in (2, 7) moved to (2, 6).
[741] Normal zombie in (3, 7) moved to (3, 6).
[741] Normal zombie in (4, 7) moved to (4, 6).
[747] Normal zombie in (0, 9) moved to (0, 8).
[747] Normal zombie in (1, 9) moved to (1, 8).
[747] Normal zombie in (2, 9) moved to (2, 8).
[747] Normal zombie in (3, 9) moved to (3, 8).
[747] Normal zombie in (4, 9) moved to (4, 8).
[788] Normal zombie in (0, 6) moved to (0, 5).
[788] Normal zombie in (1, 6) moved to (1, 5).
[788] Normal zombie in (2, 6) moved to (2, 5).
[788] Normal zombie in (3, 6) moved to (3, 5).
[788] Normal zombie in (4, 6) moved to (4, 5).
[794] Normal zombie in (0, 8) moved to (0, 7).
[794] Normal zombie in (1, 8) moved to (1, 7).
[794] Normal zombie in (2, 8) moved to (2, 7).
[794] Normal zombie in (3, 8) moved to (3, 7).
[794] Normal zombie in (4, 8) moved to (4, 7).
[800] Sun autospawned. Total: 250.
[835] Normal zombie in (0, 5) moved to (0, 4).
[835] Normal zombie in (1, 5) moved to (1, 4).
[835] Normal zombie in (2, 5) moved to (2, 4).
[835] Normal zombie in (3, 5) moved to (3, 4).
[835] Normal zombie in (4, 5) moved to (4, 4).
[841] Normal zombie in (0, 7) moved to (0, 6).
[841] Normal zombie in (1, 7) moved to (1, 6).
[841] Normal zombie in (2, 7) moved to (2, 6).
[841] Normal zombie in (3, 7) moved to (3, 6).
[841] Normal zombie in (4, 7) moved to (4, 6).
[882] Normal zombie in (0, 4) moved to (0, 3).
[882] Normal zombie in (1, 4) moved to (1, 3).
[882] Normal zombie in (2, 4) moved to (2, 3).
[882] Normal zombie in (3, 4) moved to (3, 3).
[882] Normal zombie in (4, 4) moved to (4, 3).
[888] Normal zombie in (0, 6) moved to (0, 5).
[888] Normal zombie in (1, 6) moved to (1, 5).
[888] Normal zombie in (2, 6) moved to (2, 5).
[888] Normal zombie in (3, 6) moved to (3, 5).
[888] Normal zombie in (4, 6) moved to (4, 5).
[900] Sun autospawned. Total: 275.
[929] Normal zombie in (0, 3) moved to (0, 2).
[929] Normal zombie in (1, 3) moved to (1, 2).
[929] Normal zombie in (2, 3) moved to (2, 2).
[929] Normal zombie in (3, 3) moved to (3, 2).
[929] Normal zombie in (4, 3) moved to (4, 2).
[935] Normal zombie in (0, 5) moved to (0, 4).
[935] Normal zombie in (1, 5) moved to (1, 4).
[935] Normal zombie in (2, 5) moved to (2, 4).
[935] Normal zombie in (3, 5) moved to (3, 4).
[935] Normal zombie in (4, 5) moved to (4, 4).
[976] Normal zombie in (0, 2) moved to (0, 1).
[976] Normal zombie in (1, 2) moved to (1, 1).
[976] Normal zombie in (2, 2) moved to (2, 1).
[976] Normal zombie in (3, 2) moved to (3, 1).
[976] Normal zombie in (4, 2) moved to (4, 1).
[982] Normal zombie in (0, 4) moved to (0, 3).
[982] Normal zombie in (1, 4) moved to (1, 3).
[982] Normal zombie in (2, 4) moved to (2, 3).
[982] Normal zombie in (3, 4) moved to (3, 3).
[982] Normal zombie in (4, 4) moved to (4, 3).
[1000] Sun autospawned. Total: 300.
[1023] Normal zombie in (0, 1) moved to (0, 0).
[1023] Normal zombie in (1, 1) moved to (1, 0).
[1023] Normal zombie in (2, 1) moved to (2, 0).
[1023] Normal zombie in (3, 1) moved to (3, 0).
[1023] Normal zombie in (4, 1) moved to (4, 0).
[1029] Normal zombie in (0, 3) moved to (0, 2).
[1029] Normal zombie in (1, 3) moved to (1, 2).
[1029] Normal zombie in (2, 3) moved to (2, 2).
[1029] Normal zombie in (3, 3) moved to (3, 2).
[1029] Normal zombie in (4, 3) moved to (4, 2).
[1070] Normal zombie in (0, 0) attempted entering the house.
[1070] Normal zombie in (1, 0) attempted entering the house.
[1070] Normal zombie in (2, 0) attempted entering the house.
[1070] Normal zombie in (3, 0) attempted entering the house.
[1070] Normal zombie in (4, 0) attempted entering the house.
[1070] You've lost! Zombie entered at lane 0
